st2api/st2api/BUILD

```diff
@@ -1 +1,12 @@
-python_sources()
+python_sources(
+    overrides={
+        "app.py": dict(
+            dependencies=[
+                # controller routes are wired up via st2common/st2common/openapi.yaml
+                "./controllers",
+                "./controllers/v1",
+                # "./controllers/exp",
```

Comment:
No APIs are in the experimental folder right now. So, I left this commented out.